### ISSUE TITLE
MVP

### ISSUE THREAD

**Issue by eduderewicz** [wafflehaus/chewcrew#33]
What functionality do we need to design and develop to get to MVP?

**Comment by ryyan** [comment]
- Places API Integration (Yelp/Google/etc.)
- Updated voting system (multiple choices)
- Polished UI

**Comment by eduderewicz** [comment]
@ryyan users will have to enter their name manually each time right? We have no notion of accounts for MVP correct?

**Comment by ryyan** [comment]
Correct

**Comment by eduderewicz** [comment]
I'm on board, I'll try to make some crappy sketches

**Comment by eduderewicz** [comment]
@ryyan Here's what im thinking for workflow based on our discussion the other day:
2 roles: ChewCrew creator, ChewCrew member

### Setup
- Creator setups up a vote
- Creator inputs central address and acceptable distance (0.5 mile, 5 mile, 10 mile, etc)

### Share
- ChewCrew generates a unique voting link
- Creator shares link with his/her crew

### Voting
- Creator votes on their preferred lunch category
- ChewCrew member visits link, enters name and votes on lunch category

### Decision
- Creator closes vote and the winning restaurant is revealed to all

**Comment by ryyan** [comment]
Looks good. This system does away with the Ready screen, and instead lets Members votes as soon as they join. So they only need to return once a winner is picked.

Originally went without a central Creator to keep it democratic and have the Crew be equal, but the need for a central "admin" per session becomes obvious when some people don't vote (possibly lose connection or lose the session link).

Not sure about having to pick a category. Keeping this out means one less decision to make, and gives an element of surprise. By default, the choices given will each be of a unique category (ex: 1 Italian, 1 Fast  Food, 1 Burger, etc..) and also above a certain rating (3 star). This gives users a wide enough variety and keeps it democratic through randomness.

**Comment by eduderewicz** [comment]
I agree. I don't think the Creator has an abundance of power... More like a guide or moderator to move the voting along. The Creator will have even less capability when we add accounts since we can add an "i'm in/out" flag. 

Basically, system presents categories. Voting decides most popular category, then the system randomly chooses a 3+ star place that meets the distance and category criteria.

**Comment by ryyan** [comment]
Ah gotcha, yes that system sounds bueno. Let's move forward with that.

1. Should have multiple choices of categories? Or pick just one?

2. What happens if a place is picked that nobody likes?

3. Will users eventually be able to also vote on Price/Distance/Rating, or would those remain in the Creator's hands, and everyone else just picks Category(ies)?

**Comment by eduderewicz** [comment]
1. Good point, maybe we should do pick top 3? top choice gets 3 points, 2nd 2 points, last place gets 1 point.. Category with most points wins?

2. We can have a "no way" button that anyone can select to request another option? Perhaps require Creator and/or multiple people to select "no way"

3. Sound like good features for later, maybe limit it to $$ for now by default?

**Comment by ryyan** [comment]
1. Ya 3 choices of categories sounds good. Let's make them all equal in points for the MVP, having 1st/2nd/3rd can be added in later.

2. Haha a No Way button sounds good. Dunno if it should be in MVP just yet but it's something to keep in mind. There could also be a "Let's Eat!" along with it which wouldn't do anything except show support.

3. Aye, for MVP everything will be defaulted except for: Location/Distance (picked by Creator), and Category (voted by Creator and Members)
